### PR TITLE
update pipeline version to 0.5.0

### PR DESCRIPTION
**Pull request by hongye-sun**

### CODE REVIEW

File: layouts/shortcodes/pipelines/latest-version.html [hunk 1]
```diff
@@ -1 +1 @@
-0.4.0
\ No newline at end of file
+0.5.0
```
Comment (sarahmaddox) [inline layouts/shortcodes/pipelines/latest-version.html:1]: Please would you remove the empty line at the end of the file. I think you need to do this update via the command line, as the web UI always puts the empty line back.

The empty line causes problems in rendering the version number on the page.

Comment (hongye-sun) [reply layouts/shortcodes/pipelines/latest-version.html]: Fixed, thanks.

**Pull request merged**